src/main/resources/cpp-modern-json/enum-source.mustache

```diff
@@ -24,4 +24,5 @@ void from_json(const ::nlohmann::json& j, {{classname}}& value) {
     }
 {{/enumVars}}
 {{/allowableValues}}
+    throw invalid_enum_value(str);
```

Comment:
This is a very minimalist exception message, but I guess it's clear given the type of exception